gdx/test/com/badlogic/gdx/math/PolygonTest.java

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+ * Copyright 2024 See AUTHORS file.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.math;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class PolygonTest {
+
+	@Test
+	public void testZeroRotation () {
+		float[] vertices = {0, 0, 3, 0, 3, 4};
+		Polygon polygon = new Polygon(vertices);
+		polygon.rotate(0);
+		assertArrayEquals("The polygons vertices don't correspond.", polygon.getTransformedVertices(), polygon.getVertices(), 1f);
```

Comment:
Should be `polygon's` since there is one polygon and it's possessive. Then again, the other tests that already exist in libGDX are a grammatical disaster and no-one really cares about that, since there's bigger things to worry about in life.